README.yaml

```diff
@@ -70,15 +81,14 @@ usage: |-
 
   ```hcl
   module "example" {
-    source = "https://github.com/cloudposse/terraform-example-module.git?ref=master"
-    example = "Hello world!"
+    source = "https://github.com/cloudposse/terraform-aws-guardduty.git?ref=master"
```

Comment:
Shouldn't we provide an event rule example?

https://summitroute.com/blog/2019/03/06/guardduty_event_collection_via_cloudwatch_events/

Reply:
647180542b4378772546e7e01ebb0557d941eab6